README.md

```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Engine. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-engine/en/latest/installation/index.html#using-sources).
 
-Centreon Engine needs Centreon Clib to be build. You should
-[install it first](https://github.com/centreon/centreon-clib).
+
+"""
+()[For Centos 7]() :
+First of all, check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+
+    git, make, cmake.
+
+If they are not installed, pleade install them.
```

Comment:
pleade ??

Reply:
You have to specify how to install conan.
For centos7, do prefer pip3 install conan
For other OS, do prefer the one given by the OS package manager